x-pack/plugins/apm/public/components/app/RumDashboard/index.tsx

```diff
@@ -24,12 +32,42 @@ export function RumOverview() {
     return config;
   }, []);
 
+  const {
+    urlParams: { start, end },
+  } = useUrlParams();
+
+  const { data } = useFetcher(
+    (callApmApi) => {
+      if (start && end) {
+        return callApmApi({
+          pathname: '/api/apm/services',
+          params: {
+            query: {
+              start,
+              end,
+              uiFilters: JSON.stringify({ agentName: ['rum-js'] }),
```

Comment:
Older agent versions used "js-base", so we'll need to include that if we support showing their data here.

Reply:
Maybe we should add:
```js
const RUM_SERVICES = ['js-base', 'rum-js']
```

to x-pack/plugins/apm/common/agent_name.ts. And then update `isRumAgentName` to use that instead:
```diff
-return agentName === 'js-base' || agentName === 'rum-js';
+return RUM_SERVICES.includes(agentName);
```

Reply:
That type of constant should probably go in [common/agent_name.ts](https://github.com/elastic/kibana/blob/49ef80501aef4962da2f8d82c31f869db5c03211/x-pack/plugins/apm/common/agent_name.ts).